sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala

```diff
@@ -3392,4 +3391,20 @@ class AstBuilder(conf: SQLConf) extends SqlBaseBaseVisitor[AnyRef] with Logging
     val functionName = Option(ctx.multipartIdentifier).map(visitMultipartIdentifier)
     ShowFunctionsStatement(userScope, systemScope, pattern, functionName)
   }
+
+  /**
+   * Create a [[DropFunctionCommand]] statement.
```

Comment:
`DropFunctionCommand` -> `DropFunctionStatement`?

Reply:
`Create a plan for a DROP FUNCTION statement.` will be better here.

Reply:
Yes, thanks!